.github/scripts/enable-ccache.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -e # exit on error
+set -x # echo on
+set -o pipefail # fail of any command in pipeline is an error
+
+if [ -z "$GITHUB_WORKSPACE" ]; then
+  DIR=`pwd`
+else
+  DIR=`cygpath "$GITHUB_WORKSPACE"`
+fi
+
+pushd /
+  echo "::group::/etc/makepkg.conf"
+    patch -p1 -b -i "$DIR/patches/ccache/0001-makepkg.patch"
+    cat /etc/makepkg.conf
```

Comment:
why is it needed?

Reply:
Pacman build system, `makepkg` uses global "options" to change what "features" should be enabled during build of packages. This will enable ccache globally for building all pacman packages that do not explicitly disable it.

Reply:
The question was about this line
`cat /etc/makepkg.conf`

Reply:
To see other build related variable for build failures debugging purposes.